amaznot/app/Http/Controllers/RegisterController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\Hash;
+
+class RegisterController extends Controller
+{
+    public function index()
+    {
+        return view('auth.register');
+    }
+
+    public function store(Request $request)
+    {   
+        // Validate
+        $this->validate($request,
+        [
+            'name' => 'required|unique:users',
+            'password' => 'required|confirmed',
+        ]);
+
+        // Store new user
+        User::create([
+            'name' => $request->name,
+            'role' => 'user',
+            'password' => Hash::make($request->password),
+        ]);
+
+        // Sign in User
+        auth()->attempt($request-> only('email', 'password'));
```

Comment:
You are calling the "only" method with the email. Did you mean to use the name here?

Reply:
yeah